app/models/utility/cell_count_spot_checking.rb

```diff
@@ -51,7 +52,7 @@ def second_replicates
     #
     # @return [Array<Well>] the wells in the plate that are not empty
     def filtered_wells
-      @filtered_wells ||= plate.wells_in_columns.reject(&:empty?)
+      @filtered_wells ||= plate.wells.reject(&:empty?).sort_by(&:coordinate)
```

Comment:
does that way of sorting work ok? still in columns and not A11 not before A2?
there's also `wells_with_aliquots` which does something similar